lib/system/file_test.go

```diff
@@ -0,0 +1,142 @@
+package system
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+// populates a test workspace (tmp) with a src containing several files
+func createSourceDir(t *testing.T) (tmp, src string, fcount int, err error) {
```

Comment:
This function never returns an error.  In all error paths, it uses `t.Skipf` or some other form of abort.  Thus we can drop the extra error parameter.

Reply:
Addressed in https://github.com/gravitational/robotest/pull/174/commits/66443c17c4e544f6433a34dc3acfe67927f343cc